src/logic/translations.js

```diff
@@ -0,0 +1,229 @@
+const language_info = {
+  br: {
+    original_name: 'Brezhoneg',
+    english_name: 'Breton',
+  },
+  ca: {
+    original_name: 'Català',
+    english_name: 'Catalan',
+  },
+  en: {
+    original_name: 'English',
+    english_name: 'English',
+  },
+  eo: {
+    original_name: 'Esperanto',
+    english_name: 'Esperanto',
+  },
+  fr: {
+    original_name: 'Français',
+    english_name: 'French',
+  },
+  galo: {
+    original_name: 'Galo',
+    english_name: 'Gallo',
+  },
+};
+
+const texts = {
+  en: {
+    eventLabel: 'Event',
+    round: 'Round',
+    group: 'Group',
+    station: 'Station',
+    name: 'Name',
+    new_competitor: 'New competitor',
+    scr: 'Scr',
+    result: 'Result',
+    judge: 'Judge',
+    comp: 'Comp',
+    extra: 'Extra attempt',
+    delegate_initials: 'Delegate initials',
+    cutoff: 'Cutoff',
+    time_limit: 'Time limit',
+    global: 'global',
```

Comment:
I think "total" is much more clear than "global", since it indicates summing numbers. But, I think it's fine to omit the "in" and "for" that we currently have.

```suggestion
    total: 'total',
```